Makefile

```diff
@@ -131,3 +131,41 @@ build-docker-develop:
 	docker build -t "maticnetwork/heimdall:develop" -f docker/Dockerfile.develop .
 
 .PHONY: contracts build
+
+PACKAGE_NAME          := github.com/maticnetwork/heimdall
+GOLANG_CROSS_VERSION  ?= v1.17.3
```

Comment:
Any issues if a server is using a different minor version of go?

Reply:
This only applies to the CI used to build the bins, the idea is to stop installing Go and building the binaries on the servers to move towards immutability.